commands/certmanager/delete.go

```diff
@@ -46,36 +48,42 @@ func CertDeleteCmd() *core.Command {
 
 func CmdDelete(c *core.CommandConfig) error {
 	var err error
+
 	allFlag, err := c.Command.Command.Flags().GetBool(constants.ArgAll)
 	if err != nil {
 		return err
 	}
+
 	if allFlag {
-		c.Printer.Verbose("Deleting all Certificates...")
+		fmt.Fprintf(c.Command.Command.ErrOrStderr(), jsontabwriter.GenerateVerboseOutput("Deleting all Certificates..."))
+
 		certs, _, err := c.CertificateManagerServices.Certs().List()
 		if err != nil {
 			return err
 		}
+
 		for _, cert := range *certs.Items {
 			msg := fmt.Sprintf("delete Certificate ID: %s", *cert.Id)
-			if err := utils.AskForConfirm(c.Stdin, c.Printer, msg); err != nil {
-				return err
+			if !confirm.FAsk(c.Command.Command.InOrStdin(), msg, viper.GetBool(constants.ArgForce)) {
+				return nil
```

Comment:
Is this intended? i.e. no longer throwing an error if the user refuses the confirmation

Reply:
yes, it was intentional given that `confirm.FAsk` never throws an error (even when the user inputs gibberish instead of yes/no). i figured it was intentionally thought out to be that way, but it would be easy to add a check for invalid input and have it return an error. the return value for when `no` should still be `nil` though.

Reply:
well, initially the CLI would throw errors if the user inputs gibberish / no. `user denied confirmation`

However, I don't mind this change, because:
1. These errors are not really part of any contract / promise
2. These errors are not intended for software integrations, etc, since those users generally want to use `-f/--force` and never have to deal with them
3. I can't think of any use case in which the user would want to parse the resulting output of the API in case he answers `no` upon changing his mind about the deletion / accidentally running the command.

So, I don't necessarily like this change, but I'm okay with it - just have to adjust the other places where an err is returned and return nil instead.

Reply:
Since you're not 100% on board with this change, I will make it have the same behavior as before (error on no/gibberish), just so we are consistent and we don't cause any potential breaking changes for users (I don't know who would parse the error output in this case, but it's better to be safe than sorry). :))))